pkg/registry/server/server.go

```diff
@@ -354,6 +354,28 @@ func ResolveAndMergeConfigsAndInjectTest(configs Getter, resolver Resolver, reso
 				image.Ref = ref
 				mergedConfig.Images = append(mergedConfig.Images, image)
 			}
+
+			// Attempt to handle a few simple raw_step types on a best-effort basis
+			for i := range config.RawSteps {
+				rawStep := config.RawSteps[i]
+				modifiedStep := rawStep.DeepCopy()
+				if rawStep.RPMImageInjectionStepConfiguration != nil {
+					modifiedStep.RPMImageInjectionStepConfiguration.To = api.PipelineImageStreamTagReference(fmt.Sprintf("%s-%s", rawStep.RPMImageInjectionStepConfiguration.To, ref))
+					modifiedStep.RPMImageInjectionStepConfiguration.From = api.PipelineImageStreamTagReference(fmt.Sprintf("%s-%s", rawStep.RPMImageInjectionStepConfiguration.From, ref))
+				} else if rawStep.ProjectDirectoryImageBuildStepConfiguration != nil {
```

Comment:
Shouldn't be `modifiedStep.ProjectDirectoryImageBuildStepConfiguration.Ref = ref` set too?

Reply:
yes, it isn't necessary since the step is already formed, but it is better to have it set. I will update that.